charts/operator/templates/role.yaml

```diff
@@ -184,6 +186,14 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
Why this is added? I was thinking that only sidecar not the operator need Kubernetes Role definition change. Was there missing permission and this is bugfix?

Reply:
Both need permissions, not just the sidecar. Remember you can run this on the operator and on the sidecar